Engine/csound_orc.lex

```diff
@@ -177,6 +177,12 @@ SYMBOL          [\[\]+\-*/%\^\?:.,!]
 "enduntil"      { *lvalp = make_token(csound, yytext);
                   (*lvalp)->type = OD_TOKEN;
                   return OD_TOKEN; }
+"for"           { *lvalp = make_token(csound, yytext);
+                  (*lvalp)->type = FOR_TOKEN;
+                  return FOR_TOKEN; }
+"in"            { *lvalp = make_token(csound, yytext);
```

Comment:
Does this break using the "in" opcode?

Reply:
that was a concern I raised on the mailing list, but I let my trust on the tests go ahead of me, so I'll double check locally that all's well

Reply:
you were correct that it was a problem, but I have it fixed now with start-condition on "for"